experimental/envoy_proxy/scripts/delete.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+readonly EXPERIMENAL_SCRIPTS_DIR="$(dirname "$0")"
+source "$EXPERIMENAL_SCRIPTS_DIR/common.sh"
+
+# shellcheck source=scripts/gcp_common
+source "$SCRIPTS_DIR/gcp_common"
+
+# Authenticate as the service account
+gcloud auth activate-service-account \
+  --project="${GCP_PROJECT_ID}" \
+  --key-file="${GCP_ACCOUNT_FILE}"
+
+# Delete the VM instance and its associated disk.
+gcloud compute instances delete "${ENVOY_SERVER_INSTANCE_NAME}" --zone="${ENVOY_SERVER_ZONE}" --quiet
```

Comment:
Are we using VMs in addition to Cloud Run now then? Or is this deleting the Cloud Run instance? Why is there a disk associated with it though? Perhaps add some documentation?